.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Fixes #
+
+## Changes
+
+Please provide a brief description of the changes here.
+
+Note: if the PR is touching an area that is not listed in the [existing areas](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/README.md), or the area does not have sufficient [domain experts coverage](https://github.com/open-telemetry/semantic-conventions/blob/main/.github/CODEOWNERS), the PR might be tagged as [experts needed](https://github.com/open-telemetry/semantic-conventions/labels/experts%20needed) and move slowly until experts are identified.
+
+## Merge requirement checklist
+
+* [ ] [CONTRIBUTING](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
+* [ ] Appropriate `CHANGELOG.md` files updated for non-trivial changes.
```

Comment:
> Appropriate `CHANGELOG.md` files

I'm not sure what this means?

Reply:
How about this?

`CHANGELOG.md` updated for non-trivial changes.